### DISCUSSION TITLE
Scheduling concurrent sessions in the same area?

### DISCUSSION THREAD

**Discussion by paulmelis** [pretalx/pretalx#1760] [Q&A] [answered]
At our event we will have 20+ booths, showcasing different technical demos. We have separate sessions in pretalx for each demo, where session holders can nicely update the details, their bios, etc. 

However, how to get these sessions into the schedule isn't entirely obvious to me. I don't want to assign non-overlapping time slots to them (in the same area/"room"), as that might suggest to the audience that the demos are time-limited, which they are not, all of them are available in the full demo time slot of a couple of hours. 

But I also don't want to create 20+ separate demo rooms, scheduling them concurrently, as that will explode the schedule view. 

Nor do I want to make a single top-level demo session and paste the demo details in it, as that loses any updates speakers make themselves and leads to an enormous session description.

Any tips how to handle this?

**Comment by rixx** [comment, marked as answer]
I'm afraid pretalx currently doesn't have a good mode to display events like this – we've seen the same problem with poster sessions, which would require the same kind of aggregate /super-session. You'll probably want to keep an eye on #1692, which is the tracking issue for this.
